wear/src/main/AndroidManifest.xml

```diff
@@ -122,32 +123,19 @@
 
             <meta-data android:name="androidx.wear.tiles.PREVIEW"
                 android:resource="@drawable/favorite_entities_tile_example" />
-        </service>
-        <service
-            android:name=".tiles.ShortcutsTile2"
-            android:label="@string/shortcuts_tile_2_title"
-            android:description="@string/shortcuts_tile_description"
-            android:permission="com.google.android.wearable.permission.BIND_TILE_PROVIDER"
-            android:exported="true">
-            <intent-filter>
-                <action android:name="androidx.wear.tiles.action.BIND_TILE_PROVIDER" />
-            </intent-filter>
 
-            <meta-data android:name="androidx.wear.tiles.PREVIEW"
-                android:resource="@drawable/favorite_entities_tile_example" />
-        </service>
-        <service
-            android:name=".tiles.ShortcutsTile3"
-            android:label="@string/shortcuts_tile_3_title"
-            android:description="@string/shortcuts_tile_description"
-            android:permission="com.google.android.wearable.permission.BIND_TILE_PROVIDER"
-            android:exported="true">
-            <intent-filter>
-                <action android:name="androidx.wear.tiles.action.BIND_TILE_PROVIDER" />
-            </intent-filter>
+            <meta-data
+                android:name="com.google.android.clockwork.tiles.MULTI_INSTANCES_SUPPORTED"
```

Comment:
wow interesting where did you find this intent/flag?

Reply:
yeah, I wanted to give a heads up about this, but I had to push my changes in a hurry.

I found this by decompiling the Google Keep Notes Wear OS app.

And I also had to upgrade the AndroidX Tiles library to an alpha version to get the `tileId` parameter. I'll investigate if I can upgrade it to an even newer version, but first I had some issues with the latest version.

Reply:
Great find! Maybe keep it as draft while you investigate upgrading to the latest version and there are still test log statements in there?

Reply:
great find and thinking!

alpha library is ok if it needs to be used, we have used alphas in the past just need to make sure they are stable enough for our use case :)

Reply:
I updated the `androidx.wear.tiles:tiles` to the latest working version, which is `1.2.0-alpha07`. The latest version is `1.2.0-beta01`, but it requires compileSdk version of API 34, and it produces the following error message:

> Dependency ‘androidx.wear.tiles:tiles:1.2.0-beta01’ requires libraries and applications that depend on it to compile against version 34 or later of the Android APIs.
> :wear is currently compiled against android-33.

And I also removed my test rubberduck logs, sorry 😆

Reply:
FYI the new `tileId` parameter (which is required for our use case) has been introduced in the `1.2.0-alpha05` version.

Reply:
> but it requires compileSdk version of API 34

if its required we can always bump and just test the different pieces of teh app to make sure its still working, compile SDK should have less of an impact. I am actually also looking into a different library that wants to bump compile SDK. That also means we need gradle bumped which is going to happen in #3713  

I assume this will be the trend with library updates going forward :)

Reply:
So what do you suggest? When and where to do the `compileSdk` version bump? In this PR or another? If in another, before or after this PR?

Reply:
So its best for us to keep all of the modules in sync when making these changes, if its required for your feature you can bump it in this PR. Just spot check the phone/automotive module to make sure nothing breaks and it builds/runs properly. Most of our dependencies are up to date so I don't think there will be an issue just always good to double check :)

Reply:
It's not required by the feature to bump it here, so I'd rather not do it in this PR.